bin/mas-devops-notify-slack

```diff
@@ -15,19 +15,39 @@ import os
 import sys
 from mas.devops.slack import SlackUtil
 
-
-def notifyProvisionFyre(channel: str, rc: int) -> bool:
-    name = os.getenv("CLUSTER_NAME", None)
-    if name is None:
+def _getClusterName() -> str:
+    name = os.getenv("CLUSTER_NAME", "")
+    if name == "":
         print("CLUSTER_NAME env var must be set")
         sys.exit(1)
+    return name
 
-    # Support optional metadata from standard IBM CD Toolchains environment variables
-    toolchainLink = ""
+def _getToolchainLink() -> str:
     toolchainUrl = os.getenv("TOOLCHAIN_PIPELINERUN_URL", None)
     toolchainTriggerName = os.getenv("TOOLCHAIN_TRIGGER_NAME", None)
     if toolchainUrl is not None and toolchainTriggerName is not None:
-        toolchainLink = f" | <{toolchainUrl}|Pipeline Run>"
+        toolchainLink = f"<{toolchainUrl}|{toolchainTriggerName}>"
+        return toolchainLink
+    return ""
+
+def _broadcast(channels: str, message: list):
+    channelList = [ch.strip() for ch in channels.split(",")]
+    responses = []
+    for channel in channelList:
+        try:
+            if channel:
+                response = SlackUtil.postMessageBlocks(channel, message)
+                responses.append(response.data.get("ok", False))
+        except Exception as e:
+            print(f"Failed to send message to channel {channel}: {e}")
+            responses.append(False)
+    return all(responses) if responses else True
```

Comment:
responses is a list ... "if responses` isn't natural to read ... what's the intent here?  If responses is an empty list, if it's None/undefined - in either case why would we return True in that case (assuming true/false return value indicates success, wouldn't either scenario indicate something went wrong?